packages/modules-sdk/src/types/module.ts

```diff
@@ -0,0 +1,103 @@
+import { AwilixContainer } from "awilix"
+import { Logger as _Logger } from "winston"
+
+export type LogLevel =
+  | "query"
+  | "schema"
+  | "error"
+  | "warn"
+  | "info"
+  | "log"
+  | "migration"
+export type LoggerOptions = boolean | "all" | LogLevel[]
+
+export type ClassConstructor<T> = {
+  new (...args: unknown[]): T
+}
+
+export type MedusaContainer = AwilixContainer & {
```

Comment:
can't wait for the types to be separated 😂 here if medusa signatures change for that type it will not work anymore, and that goes for all sort of types